.gitignore

```diff
@@ -8,3 +8,4 @@ proto/build
 !/drizzle/**/*.sql
 .eslintcache
 docs/api/html/*
+test/fixtures/config/*.zip
```

Comment:
What happens to all the existing fixtures when we do this?

Reply:
Before merging I'll `git rm` all the zips, so with this they should be untracked, right?

Reply:
Yeah. Could you do that before we merge?